test/bin/build_images.sh

```diff
@@ -25,6 +25,29 @@ osbuild_logs() {
     done
 }
 
+extract_container_images() {
+    local -r version=$1
+    local -r repo_url=$2
+    local -r outfile=$3
+
+    mkdir -p "${IMAGEDIR}/release-info-rpms"
+    pushd "${IMAGEDIR}/release-info-rpms"
+    sudo bash -c 'cat > /etc/yum.repos.d/microshift.repo' << EOF
+[microshift]
+name="microshift ${version}"
+baseurl="${repo_url}"
+enabled=1
+check_gpg=false
```

Comment:
Why `false`? I think we can enable gpg checks here.

Reply:
Was just following the trend with the repos in https://github.com/openshift/microshift/tree/main/test/package-sources, all of them have the gpg disabled because half are local and hold no configuration for it.